source/extensions/filters/network/zookeeper_proxy/filter.cc

```diff
@@ -216,13 +244,51 @@ void ZooKeeperFilter::onDecodeError() {
   setDynamicMetadata("opname", "error");
 }
 
-void ZooKeeperFilter::onRequestBytes(const uint64_t bytes) {
+void ZooKeeperFilter::onDefaultRequestBytes(const OpCodes opcode, const uint64_t bytes) {
+  auto it = config_->op_code_map_.find(opcode);
+  ASSERT(it != config_->op_code_map_.end());
+  const ZooKeeperFilterConfig::OpCodeInfo& opcode_info = it->second;
```

Comment:
probably 
```
ASSERT(config_->op_code_map_.contains(opcode));
config_->op_code_map_[opcode].rq_bytes_counter_add(bytes);
```
wdyt?

Reply:
Thanks, it is much more clean, will make the change accordingly.